app/views/layouts/_header.html.erb

```diff
@@ -8,4 +8,7 @@
   <a href="/notechforice">
     <h2>Why No Tech For ICE</h2>
   </a>
+  <a href="/branding">
+    <h2>Branding Info</h2>
+  </a>
```

Comment:
Should we use the built in url helpers for these or is the goal to make it easier to transition to a frontend client?

Reply:
There wasn't a specific goal one way or the other, I think I just used this bc I forget the fiddly bits in the erb helpers sometimes/often. Do we need these static links to be helpers when it's just static data?

(this is a sincere question--I actually don't know why you would choose one over the other; this just seems like less processing power and also less human work)